docs/docs/cmd/teams/conversationmember/conversationmember-add.md

```diff
@@ -0,0 +1,54 @@
+# teams conversationmember add
+
+Adds or sets a conversationmember in a private channel
+
+## Usage
+
+```sh
+m365 teams conversationmember add [options]
+```
+
+## Options
+
+`-i, --teamId [teamId]`
+: The ID of the team where the channel is located. Specify either `teamId` or `teamName`, but not both.
+
+`--teamName [teamName]`
+: The name of the team where the channel is located. Specify either `teamId` or `teamName`, but not both.
+
+`-c, --channelId [channelId]`
+: The private channel's ID. Specify either `channelId` or `channelName`, but not both.
+
+`--channelName [channelName]`
+: The private channel's name. Specify either `channelId` or `channelName`, but not both.
+
+`--userId [userId]`
+: The user's ID or userprincipalname. You can also pass a comma separated list of userIds.
+
+`--userDisplayName [userDisplayName]`
+: The displayName of a user. You can also pass a comma separated list of displaynames.
+
+`--owner [owner]`
```

Comment:
Following the spec, `owner` was supposed to be used as a flag without accepting any value so it should be defined as `--owner` rather than `--owner [owner]` which expects a value.